### PR TITLE
feat(lottery): add draw info box

### PR DESCRIPTION
**Pull request by ghader-Salehi**

### CODE REVIEW

File: src/components/Svg/index.js [hunk 1]
```diff
@@ -9,3 +9,6 @@ export { default as SvgTokenPocket } from './TokenPocket'
 export { default as SvgTrustWallet } from './TrustWallet'
 export { default as SvgUser } from './User'
 export { default as SvgWalletConnect } from './WalletConnect'
+export { default as SvgPrevious } from './Previous'
+export { default as SvgNext } from './Next'
+export { default as SvgEnd } from './End'
```
Comment (parsaakbari1209) [inline src/components/Svg/index.js:14]: Please sort these, it makes finding them much easier 👯

File: src/components/View/Lottery/components/DrawInfo/Box.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react'
+import styled from 'styled-components'
+
+import { Flex } from '@/components/Core/Toolkit'
+
+import { SvgSolidChevronDown } from '@/components/Svg'
```
Comment (parsaakbari1209) [inline src/components/View/Lottery/components/DrawInfo/Box.js:6]: The way I sort the imports is as follows:
```suggestion
// The first group is nextjs and react internal packages.
import { useState } from 'react'

// The second group is external packages.
import styled from 'styled-components'

// And the third group is internal packages.
import { Flex } from '@/components/Core/Toolkit'
import { SvgSolidChevronDown } from '@/components/Svg'
```

File: src/components/View/Lottery/components/DrawInfo/Content.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
```
Comment (parsaakbari1209) [inline src/components/View/Lottery/components/DrawInfo/Content.js:1]: Is there a reason for importing `React`? 👀

File: src/components/View/Lottery/components/DrawInfo/Detail.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
```
Comment (parsaakbari1209) [inline src/components/View/Lottery/components/DrawInfo/Detail.js:1]: Is there a reason for importing React? 👀

File: src/components/View/Lottery/components/DrawInfo/Content.js (hunk 3 above)
Comment (ghader-Salehi) [reply src/components/View/Lottery/components/DrawInfo/Content.js:1]: Sorry i forgot to delete it

File: src/components/View/Lottery/components/DrawInfo/Detail.js (hunk 4 above)
Comment (ghader-Salehi) [reply src/components/View/Lottery/components/DrawInfo/Detail.js:1]: i forgot this one also :)

File: src/components/View/Lottery/components/DrawInfo/Content.js (hunk 3 above)
Comment (parsaakbari1209) [reply src/components/View/Lottery/components/DrawInfo/Content.js:1]: no problemo 💯

**Review by parsaakbari1209 — APPROVED** [review]
Looks Awesome 🌟

**Pull request merged**